modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-11-0_{context}"]
+= Release notes for {gitops-title} 1.11.0
+
+{gitops-title} 1.11.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+The current release adds the following improvement:
+
+* With this update, you can selectively disable the `redis` and `application-controller` components for an Argo CD instance in a specified namespace. These components are enabled by default.
+To disable a component, set the `enabled` flag to `false` in the `.spec.<component>.enabled` field of the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-3723[GITOPS-3723]
++
+.Example: Disabling the `redis` and `application-controller` components
+[source, yaml]
+----
```

Comment:
I suggest to consider adding initial fields to this YAML. For example, some more context like: `apiVersion: argoproj.io/v1alpha1
kind: ArgoCD
metadata:
  name: example-argocd`

Reply:
ACK.

Addressed.